src/m.graph.js

```diff
@@ -432,11 +433,13 @@ please.GraphNode = function () {
         please.make_animatable(
             this, "billboard_mode", this.__billboard_driver, this.shader, true);
 
-        // prog.samplers is a subset of prog.vars
-        for (var name, i=0; i<prog.uniform_list.length; i+=1) {
-            name = prog.uniform_list[i];
-            if (ignore.indexOf(name) === -1 && !this.shader.hasOwnProperty(name)) {
-                please.make_animatable(this, name, null, this.shader);
+        if (please.gl.ctx !== null) {
```

Comment:
I'm unsure if behavior that is specific to 2D mode should be inside a function called "regen_glsl_bindings".  Maybe some of the functionality in this function should be folded out into another method.